vendor/bat-native-ads/src/bat/ads/internal/account/account_unittest.cc

```diff
@@ -584,11 +589,13 @@ TEST_F(BatAdsAccountTest, DepositForNonCash) {
 
   transactions::GetForDateRange(
       DistantPast(), DistantFuture(),
-      [&expected_transactions](const bool success,
-                               const TransactionList& transactions) {
-        ASSERT_TRUE(success);
-        EXPECT_EQ(expected_transactions, transactions);
-      });
+      base::BindOnce(
+          [](TransactionList expected_transactions, const bool success,
```

Comment:
`The parameter 'expected_transactions' is copied for each invocation but only used as a const reference; consider making it a const reference`

Reply:
changed parameter type to `const TransactionList& expected_transactions`